scripts/generate-sealed-secrets-pki.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+VAULT_NAME=$1
+NAMESPACE=${2:-admin}
+
+openssl req -x509 -newkey rsa:4096 -keyout tls.key -out tls.crt -days 365 \
```

Comment:
is there any plan to check if this is in vault first and then download if not?
this script isn't currently safe to run multiple times

Reply:
this should be separated out into a separate task/task group.  We need a way to regenerate the sealed secrets PKI and then the secrets.  Think this might be a story for later (but should be done ASAP)